concent_api/core/models.py

```diff
@@ -291,6 +320,10 @@ class Meta:
 
     task_id = CharField(max_length=MESSAGE_TASK_ID_MAX_LENGTH)
 
+    computation_deadline        = DateTimeField()
+
+    result_package_size         = IntegerField()
```

Comment:
Please add a validation (inside the `clean()` method) that these fields always match the values inside `report_computed_task`.